src/components/Guides/BuildSakura/BuildSakura.js

```diff
@@ -1,98 +1,98 @@
-import React, {useEffect,useState}  from "react";
+import React, { useEffect, useState } from "react";
 import Loader from "../../Loader/Loader";
 import "./BuildSakura.css";
 import SyntaxHighLighter from "react-syntax-highlighter";
 import { dracula } from "react-syntax-highlighter/dist/cjs/styles/hljs";
 
 const BuildSakura = () => {
   const [loading, setLoading] = useState(false);
-  useEffect(()=>{
+  useEffect(() => {
     setLoading(true)
     setTimeout(() => {
       setLoading(false)
     }, 1500);
-  },[])
+  }, [])
   return (
     <div className="environment">
-       {loading && (
-    <Loader /> )}
-       {!loading && (
-      <center>
-        <div className="content">
-          <h2>BUILDING SAKURA</h2>
-          <p>
-            Here we will guide you on how to build Project Sakura for your
-            device.
+      {loading && (
+        <Loader />)}
+      {!loading && (
+        <center>
+          <div className="content">
+            <h2>BUILDING SAKURA</h2>
+            <p>
+              Here we will guide you on how to build Project Sakura for your
+              device.
           </p>
-          <hr />
-          <h2>HOW TO SETUP A BUILD ENVIRONMENT</h2>
-          <p>
-            To Build Project Sakura you can use any 64 Bit Linux Distro, though
-            we recommend using Ubuntu 18.04, LinuxMint, MXLinux or Manjaro. You
-            can setup the build environment using
-            <a href="https://github.com/akhilnarang/scripts" target="_blank">
-              {" "}
+            <hr />
+            <h2>HOW TO SETUP A BUILD ENVIRONMENT</h2>
+            <p>
+              To Build Project Sakura you can use any 64 Bit Linux Distro, though
+              we recommend using Ubuntu 18.04, LinuxMint, MXLinux or Manjaro. You
+              can setup the build environment using
+            <a href="https://github.com/akhilnarang/scripts" target="_blank" rel="noreferrer">
```

Comment:
can you tell me what rel="noreferrer" doing i am intrested to know

BTW the PR looks good to me
Just one advice can you not include the indentation changes in the next PR, its hard to review @Samridhi-98 

Nevertheless solid work

Reply:
` rel="noreferrer"` is basically used to protect the user from hijacking while browsing your site as links to cross-origin destinations are unsafe.
That indentation occurs due to my vscode extension, I will disable it while working on sakura if you say so.